storybook/src/components/DescriptionList/DescriptionList.stories.tsx

```diff
@@ -33,16 +35,48 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {}
 
-export const MultipleDetails: Story = {
+export const MultipleDescriptions: Story = {
+  args: {
+    children: [
+      <DescriptionList.Term key={1}>blinde, slechtziende</DescriptionList.Term>,
```

Comment:
I think you've changed the convention here, and I'm not sure that's what we want. Previously, we always started terms and descriptions with a capital, in these examples you sometimes do and sometimes don't.

I think starting with a capital is more visually appealing.

Reply:
I don’t think we had that convention in place. The first two examples concern phrases that often sit in the middle of a sentence. They will not be capitalized there, which is a good reason not to do it here.

The [source document](https://www.amsterdam.nl/schrijfwijzer/inclusieve-taal-richtlijnen-tips/inclusieve-woordenlijst/), written by content specialists, uses lower case as well.

The third example uses brand names and complete sentences, thus capital letters and punctuation.

Reply:
Maybe we should specify or link to the convention (in a separate PR)? [The convention for print is a bit simpler](https://amsterdam.nl/stijlweb/basiselementen/typografie/#hb251a4e3-5384-43be-a7ee-829d14c49fea), and leads to a more appealing result, imo.

Wasn't there anything on this on the old Stijlweb site?

Reply:
Well, look at that:

> losse woorden, grammaticaal complete en incomplete zinnen worden met beginkapitaal gezet

Doesn’t sound like this is specific for print, so I’ve changed them.

Reply:
Created [a new epic](https://gemeente-amsterdam.atlassian.net/browse/DES-1008) to keep track of the content guidelines we’ll once add to our docs.